opentracing-api/pom.xml

```diff
@@ -25,8 +25,8 @@
         <version>0.1.0-SNAPSHOT</version>
     </parent>
 
-    <artifactId>opentracing</artifactId>
-    <name>OpenTracing</name>
+    <artifactId>opentracing-api</artifactId>
```

Comment:
sorry that I'm so ignorant about this -- but can you very briefly educate me about the impact of this change as far as Maven is concerned? I'm guessing that people who depend on the Java OT API (as callers) will list `"opentracing-api"` rather than `"opentracing"`, correct?